components.yaml

```
@@ -5,54 +5,54 @@ GEOSadas:
 env:
   local: ./@env
   remote: ../ESMA_env.git
-  tag: v1.4.3
+  tag: v3.2.1
   develop: main
 
 cmake:
   local: ./@cmake
   remote: ../ESMA_cmake.git
-  tag: v1.0.11
+  tag: v3.4.2
   develop: develop
 
 ecbuild:
   local: ./@cmake/@ecbuild
   remote: ../ecbuild.git
-  tag: geos/v1.0.0
+  tag: geos/v1.0.6
 
 NCEP_Shared:
   local: ./src/Shared/@NCEP_Shared
   remote: ../NCEP_Shared.git
-  tag: v1.1.0
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: main
 
 GMAO_Shared:
   local: ./src/Shared/@GMAO_Shared
   remote: ../GMAO_Shared.git
-  tag: cvs/GEOSadas-5_27_1_p3
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: dev/adas
 
 MAPL:
   local: ./src/Shared/@MAPL
   remote: ../MAPL.git
-  tag: v1.1.14
+  tag: v2.6.7
   develop: develop
 
 FMS:
   local: ./src/Shared/@FMS
   remote: ../FMS.git
-  tag: geos/orphan/v1.0.3
+  tag: geos/2019.01.02+noaff.7 
   develop: geos/release/2019.01
 
 GEOSana_GridComp:
   local: ./src/Components/@GEOSana_GridComp
   remote: ../GEOSana_GridComp.git
-  tag: v1.3.0
+  tag: rt-GEOSadas-5_29_0_prep3
```

```suggestion
  tag: v1.4.0
```

This should now be v1.4.0 as that is the equivalent to the rt-GEOSadas-5_29_0_prep4 tag:

https://github.com/GEOS-ESM/GEOSana_GridComp/compare/v1.4.0..rt-GEOSadas-5_29_0_prep4

---

The changes to anaGridComp seem fine to me; except that since this was introduced we now have PR#44 that would be nice to accommodate here.

---

I've released v1.4.1 of GEOSana_GridComp with the metop-b fix.